.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,39 @@
+# https://github.com/golangci/golangci-lint-action#how-to-use
+name: golangci-lint
+on:
+  push:
+    branches: [main]
+  pull_request:
+permissions:
+  contents: read
+  pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      # this is needed by containers/storage which we're using in gangplank
```

Comment:
(Which we should definitely stop vendoring at some point; we should not embed a container runtime, but instead talk to one via the kube API or `podman --remote`)

Reply:
Yeah... I hadn't realized we were vendoring it until working on this. Was quite the surprise.